decidim-verifications/lib/decidim/verifications/workflow_manifest.rb

```diff
@@ -34,7 +34,8 @@ class WorkflowManifest
       attribute :form, String
       attribute :expires_in, ActiveSupport::Duration, default: 0.minutes
       attribute :action_authorizer, String
-
+      attribute :renewable, Boolean, default: false
+      attribute :time_between_renews, ActiveSupport::Duration, default: 0.minutes
```

Comment:
Maybe we should use the same value that is in [the generator template](https://github.com/decidim/decidim/pull/5854/commits/1db99f678ee511733dd4e41d4f299a97fe16768b#diff-62036b46ba53aaf7aa8886b9eb930ab1R8) (1 day according to @andreslucena request) to avoid accidentally activating instantaneous renewals?

Reply:
Yes! good suggestion! I'll update to default `1.day`